lambda/latest_image_tracker/template.yml

```diff
@@ -0,0 +1,51 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: "AWS::Serverless-2016-10-31"
+Description: Resources for Lambda function to update SSM Parameter
+
+Resources:
+  ImageBuilderSNSTopic:
+    Type: "AWS::SNS::Topic"
+    Properties:
+      DisplayName: "Image Builder"
+
+  UpdateSSMParameter:
+    Type: "AWS::Serverless::Function"
+    Properties:
+      Handler: image-builder-lambda-update-ssm.lambda_handler
+      Runtime: python3.7
+      CodeUri: .
+      Description: Update SSM Parameter with the latest AMI
+      MemorySize: 256
+      Timeout: 300
+      AutoPublishAlias: live
+      Events:
+        SNSTopicEvent:
+          Type: SNS
+          Properties:
+            Topic: !Ref ImageBuilderSNSTopic
+      Policies:
+          - arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
```

Comment:
For AWS China and AWS GovCloud (US) compatibility, use the CloudFormation partition pseudo parameter, example: arn:${AWS::Partition}:

Reply:
Updated template.yaml to include arn:${AWS::Partition}: